_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SQL Workshop"
+description: "SQL Workshop Intro"
+date: 2019-07-25T13:24:17-07:00
+prereq: "None"
+difficulty: "Beginner"
+image: ""
+draft: false
+# icon: "fab fa-python"
+weight: 130
+# translationKey: "SQL-workshop-home"
+---
+
+# Introduction: Learn SQL by Saving the Galaxy!
+
+
+![Galaxy](media/Galaxy.png)
+
+<font size="5">Welcome to the Nuevo Space Station (NSS)! 
```

Comment:
Consider using a markdown header instead of an HTML tag? Maybe 

##### Welcome to the Nuevo Space Station (NSS)!